apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -102,12 +117,23 @@ if [[ -n "${LAUNCH_OPTIONS_JSON_STR}" ]]; then
   runner_flags+=("--launch_options_json_path=${LAUNCH_OPTIONS_JSON_PATH}")
 fi
 
+target_flags=()
+if [[ -n "$simulator_id" ]]; then
+  target_flags=(
+    "test"
```

Comment:
I am not entire sure here because runners are different but should this be "simulator_test" ?

Reply:
I'm not sure why but `xctestrunner` doesn't accept `--id` with `simulator_test`

Reply:
For the design of xctestrunner, the subcommand `simulator_test` is designed for creating new simulator for testing and the subcommand  `test` is for testing on existing simulator/device with passing id.